lib/web/JsonpChunkLoadingRuntimeModule.js

```diff
@@ -176,7 +176,7 @@ class JsonpChunkLoadingRuntimeModule extends RuntimeModule {
 						]),
 						"}",
 						prefetchChunks && prefetchChunks.length > 0
-							? prefetchChunks.map(c => `\nprefetchChunk(${c});`)
+							? prefetchChunks.map(c => `prefetchChunk("${c}");`).join("\n")
```

Comment:
I think it might be safer to do `JSON.stringify(c)` rather than `"${c}"`.

Reply:
Yep could you change that